frontend/ui/widget/numberpickerwidget.lua

```diff
@@ -126,7 +126,7 @@ function NumberPickerWidget:init()
         callback_input = function()
             input_dialog = InputDialog:new{
                 title = _("Enter number"),
-                input = self.formatted_value,
+                input_hint = T("(%1 - %2)", self.value_min, self.value_max),
```

Comment:
Not showing the current value as the starting point - to show the min/max as a hint - not sure which is best (feels like I have hated in the past some software that didn't show the current value and started from blank in case like this :)
May be include the current value in the hint ?

Reply:
> feels like I have hated in the past some software that didn't show the current value and started from blank in case like this 

+1

Reply:
Just display it in the description.

Reply:
The range, that is. Not the current value.